app/views/layouts/catalog_result.html.erb

```diff
@@ -6,6 +6,21 @@
   <section class="<%= show_content_classes %>">
     <%= yield %>
   </section>
+
+  <section class="col-lg-5">
+    <% if @document.file_format.include?('JPEG') && !@document[Settings.FIELDS[:GEOREFERENCED]] %>
+      <div role="alert" class="alert alert-info d-flex shadow-sm align-items-center">
+      <div class="bi bi-info-circle-fill fs-3 me-3 align-self-center
+                  d-flex justify-content-center"></div>
+      <div class="text-body">
+        <div>This map is not georeferenced.</div>
```

Comment:
I know we don't do it everywhere, but imo it's good practice to put these strings in the locale file, at least for new content/templates

Reply:
I removed the georeference alert. Making new branch.